README.md

```diff
@@ -1,29 +1,15 @@
 # AWS IoT Device SDK for C++ v2
 
-This document provides information about the AWS IoT device SDK for C++ V2.
-
-If you have any issues or feature requests, please file an issue or pull request.
-
-This SDK is built on the AWS Common Runtime, a collection of libraries
-([aws-c-common](https://github.com/awslabs/aws-c-common),
-[aws-c-io](https://github.com/awslabs/aws-c-io),
-[aws-c-mqtt](https://github.com/awslabs/aws-c-mqtt),
-[aws-c-http](https://github.com/awslabs/aws-c-http),
-[aws-c-cal](https://github.com/awslabs/aws-c-cal),
-[aws-c-auth](https://github.com/awslabs/aws-c-auth),
-[s2n](https://github.com/awslabs/s2n)...) written in C to be
-cross-platform, high-performance, secure, and reliable. The libraries are bound
-to C++ by the [aws-crt-cpp](https://github.com/awslabs/aws-crt-cpp) package.
+This document provides information about the AWS IoT device SDK for C++ V2. This SDK is built on the [AWS Common Runtime](https://docs.aws.amazon.com/sdkref/latest/guide/common-runtime.html)
 
 __Jump To:__
 
 * [Installation](#installation)
 * [Samples](./samples)
 * [Getting Help](#getting-help)
 * [FAQ](./documents/FAQ.md)
-* [Giving Feedback and Contributions](#giving-feedback-and-contributions)
+* [API Docs](https://aws.github.io/aws-iot-device-sdk-cpp-v2/)
 * [MQTT5 User Guide](./documents/MQTT5_Userguide.md)
-* [Secure Tunnel User Guide](./documents/Secure_Tunnel_Userguide.md)
```

Comment:
Would we like to keep the link to Secure Tunnel User guide?

Reply:
+1. Alternately, we could move the secure tunnel guide elsewhere but we should have a link somewhere to it.